sites/kit.svelte.dev/src/ambient.d.ts

```diff
@@ -0,0 +1,5 @@
+// TODO: get this from @sveltejs/enhanced-img
```

Comment:
I think we just need to have an ambient `.d.ts` module inside the package that is imported from the main `types` declaration file

Reply:
So I've been told. But I must have been screwing something up when I tried. Any chance you could take a stab at it?

My kingdom for ~~a horse~~ some correctly configured types